### PR TITLE
Clean Up README After Adding Gen SSH Key For EC2

### PR DESCRIPTION
**Pull request by sethAmazon**
# Description of the issue
Integration test readme out of date

# Description of changes
Update read me

# Tests
N/A

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make linter`

### CODE REVIEW

File: integration/terraform/ec2/localstack/variables.tf [hunk 1]
```diff
@@ -5,7 +5,7 @@ variable "ec2_instance_type" {
 
 variable "ssh_key_name" {
   type    = string
-  default = "cwagent-integ-test-key"
```
Comment (SaxyPandaBear) [inline integration/terraform/ec2/localstack/variables.tf:8]: So we took out SSH access to the EC2s?

Comment (khanhntd) [reply integration/terraform/ec2/localstack/variables.tf:8]: We have a generated SSH Key if we don't provide any value to this variable `ssh_key_name`. So newcomer is still able to SSH access to the EC2 if they provide their key to this variable and the `ssh_key_value`.

**Review by khanhntd — APPROVED** [review]
LGTM.

File: integration/terraform/ec2/localstack/variables.tf (hunk 1 above)
Comment (sethAmazon) [reply integration/terraform/ec2/localstack/variables.tf:8]: This has minimum to get up and running. I can add it back as optional.

**Review by ymtaye — APPROVED** [review]

**Comment by SaxyPandaBear** [comment]
There's overlap with this and #601 and I am leaning towards focusing on that one since the changes are more robust and holistic from someone who had to set it up from scratch after all the terraform changes.

**Comment by khanhntd** [comment]
> There's overlap with this and #601 and I am leaning towards focusing on that one since the changes are more robust and holistic from someone who had to set it up from scratch after all the terraform changes.

Agreee !

**Comment by SaxyPandaBear** [comment]
Closing since #601  was merged

**Pull request closed without merging**